### ISSUE TITLE
Cert rotation integration tests for RKE2 

### ISSUE THREAD

**Issue by Sahota1225** [rancher/rancher#37816]
This is a placeholder ticket for Cert rotation which tests the following cases: 
create an integration test for rke2/k3s

**Comment by sowmyav27** [comment]
@jakefhyde @igomez06 Do we need a backport to 2.6 for this?

**Comment by igomez06** [comment]
> @jakefhyde @igomez06 Do we need a backport to 2.6 for this?

Definitely, @jakefhyde could do you back port PR? The review will be much faster this time around

**Comment by jakefhyde** [comment]
@sowmyav27 @igomez06 Can do, which milestone should it go into, v2.6.9 or v2.6.10?

**Comment by sowmyav27** [comment]
@jakefhyde 2.6.9? Since the code is already in 2.7, this specific pick must be cherry picked to 2.6
